docs/publickeys-signers.md

```diff
@@ -197,7 +197,7 @@ In order to use these keypairs as signers throughout your application, you can u
 
 ```ts
 const myKeypair = umi.eddsa.generateKeypair();
-const myKeypairSigner = createSignerFromKeypair(myKeypair);
+const myKeypairSigner = createSignerFromKeypair('eddsa',myKeypair);
```

Comment:
'eddsa' worked, too - but i agree this looks better.

Reply:
Oh no but `"eddsa"` wouldn't work when you actually try and use the created signer. You'd also get a TypeScript error because that's not what the first argument expect.

The `createSignerFromKeypair` function requires the `eddsa` interface from the context. In other words, an object that looks like `{ eddsa: EddsaInterface }`. Passing a string would be invalid. Passing `umi` works because it extends the entire `Context` type that contains the `eddsa` interface.

https://github.com/metaplex-foundation/umi/blob/eda8af5136a4e642cf6b4a7cbb7405ae405a64a0/packages/umi/src/Keypair.ts#L35